src/lambda.js

```diff
@@ -82,8 +84,8 @@ class LambdaRunner {
       console.log('container output was:\n', output);
       console.log('container error was:\n', stderr.toString('utf8').trim());
     }
-
-    return JSON.parse(result);
+    // istanbul ignore next
+    return JSON.parse(result || '{}');
```

Comment:
I think this can be reverted if you rebase on master

Reply:
I'm not sure what you mean.  I have had times where something is wrong with the docker image, and it fails to run.  When that happens it has no output, and this throws an exception that `result` is undefined.